beetsplug/info.py

```diff
@@ -98,8 +98,9 @@ def print_data(data, item=None, fmt=None):
     formatted = {}
     for key, value in data.items():
         if isinstance(value, list):
-            formatted[key] = u'; '.join(value)
-        if value is not None:
+            if value:
+                formatted[key] = u'; '.join(value)
+        elif value is not None:
```

Comment:
This, for what it's worth, is one of the things that worries me about making arbitrary fields into lists: now client code will frequently need to have boilerplate for the form `if isinstance(value, list):`. This sort of thing puts additional weight on the side of having an explicit way to access lists, or some other foolproof conversion to produce a scalar value.